srml/contract/src/exec.rs

```diff
@@ -608,15 +610,16 @@ mod tests {
 		ExecFeeToken, ExecutionContext, Ext, Loader, EmptyOutputBuf, TransferFeeKind, TransferFeeToken,
 		Vm, VmExecResult, InstantiateReceipt, RawEvent,
 	};
-	use account_db::AccountDb;
-	use gas::GasMeter;
+	use crate::account_db::AccountDb;
+	use crate::gas::GasMeter;
+	use crate::tests::{ExtBuilder, Test};
+	use crate::{CodeHash, Config};
```

Comment:
Here it is recommended to use something like this
```rust
use crate::{
    account_db::AccountDb,
    gas::GasMeter,
    tests::{ExtBuilder, Test},
    CodeHash,
    Config,
};
```

Reply:
Obviously same for other instances of `use`.

Reply:
Yeah, i was on the fence about that! This is a new piece of syntax that arrived quite recently (and note that arrived before 2018), looking [at other PRs](https://github.com/paritytech/substrate/pull/1481/files) we don't actually use it.

So I'd happy to migrate it throughout the whole codebase if we make a decision about that.

Reply:
sounds good